AppleShopDZ/AppleShopDZ/SearchViewController.swift

```diff
@@ -0,0 +1,183 @@
+//
+//  ViewController.swift
+//  AppleShopDZ
+//
+//  Created by angelina on 06.10.2022.
+//
+
+import UIKit
+
+/// Контроллер экрана поиска
+final class SearchViewController: UIViewController {
+
+    // MARK: - Private enum
+    private enum Constants {
+        static let titleLable = "Поиск"
+        static let placeholderSearchBar = "Поиск по продуктам и магазинам"
```

Comment:
```suggestion
        static let placeholderSearchBar = "Поиск по продуктам и магазинам"
```
это точно SearchBar?